modules/client/src/tokenVoting/internal/graphql-queries/settings.ts

```diff
@@ -0,0 +1,11 @@
+import { gql } from "graphql-request";
+
+export const QueryTokenVotingPluginSettings = gql`
+query tokenVotingPluginSettings($address: ID!) {
+  tokenVotingPlugin(id: $address){
+    totalSupportThresholdPct
+    relativeSupportThresholdPct
+    minDuration
+  }
```

Comment:
Isn't the token contract address queried here?

Reply:
Nope this wasn't the case before